moduleframework/dockerlinter.py

```diff
@@ -33,10 +33,10 @@ class DockerfileLinter(object):
 
     dockerfile = None
     oc_template = None
-    dfp_structure = {}
-    docker_dict = {}
 
     def __init__(self):
+        self.dfp_structure = {}
+        self.docker_dict = {}
```

Comment:
why you moved this from previous lines here?
Is this more pythonish? I don't know, just want to know answere :-)

Reply:
There was initialization problem. `docker_dict` was not empty for some reason. Nothing else.